frontend/src/app/ml-models/ml-model-form/ml-model-form.component.ts

```diff
@@ -165,6 +164,12 @@ export class MlModelFormComponent implements OnInit {
     }
   }
 
+  get sources() {
+    const usesFirstPartyData = this.value('usesFirstPartyData');
+    let sources = Object.values(Source);
```

Comment:
Do you get errors when defining sources as a const variable as in: const sources = Object.values(Source);

I ask because 'let' is supposed to be used for variables that get reassigned and I don't see the value of 'sources' being modified.

Reply:
That's fair. I think I had originally done that because I planned to modify it (remove first party source given certain circumstances) then realized a quick ternary was simpler and still fairly clear. I've updated this to a const.